cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/runtime/artifact/ArtifactInspectorTest.java

```diff
@@ -94,26 +94,26 @@ public void testGetPluginRequirements() {
     Assert.assertTrue(artifactInspector.getPluginRequirements(InspectionApp.AppPlugin.class).isEmpty());
 
     // check that if a plugin specify a requirement it is captured
-    Assert.assertEquals(ImmutableSet.of(Requirements.TEPHRA_TX),
+    Assert.assertEquals(ImmutableSet.of(Requirements.TEPHRA_TX.toLowerCase()),
                         artifactInspector.getPluginRequirements(InspectionApp.SingleRequirementPlugin.class));
 
     // check if a plugin specify a requirement annotation but it is empty the requirement captured is no requirement
     Assert.assertTrue(artifactInspector.getPluginRequirements(InspectionApp.EmptyRequirementPlugin.class).isEmpty());
 
     // check if a plugin specify multiple requirement all of them are captured
-    Assert.assertEquals(ImmutableSet.of(Requirements.TEPHRA_TX, "secondrequirement"),
+    Assert.assertEquals(ImmutableSet.of(Requirements.TEPHRA_TX.toLowerCase(), "secondrequirement"),
                         artifactInspector.getPluginRequirements(InspectionApp.MultipleRequirementsPlugin.class));
 
     // check if a plugin has specified empty string as requirement is captured as no requirements
     Assert.assertTrue(artifactInspector
                         .getPluginRequirements(InspectionApp.SingleEmptyRequirementPlugin.class).isEmpty());
 
     // check if a plugin has specified empty string with a valid requirement only the valid requirement is captured
-    Assert.assertEquals(ImmutableSet.of(Requirements.TEPHRA_TX),
+    Assert.assertEquals(ImmutableSet.of(Requirements.TEPHRA_TX.toLowerCase()),
                         artifactInspector.getPluginRequirements(InspectionApp.ValidAndEmptyRequirementsPlugin.class));
 
     // test that duplicate requirements are only stored once and the beginning and ending white spaces are trimmed
-    Assert.assertEquals(ImmutableSet.of(Requirements.TEPHRA_TX, "duplicate"),
+    Assert.assertEquals(ImmutableSet.of(Requirements.TEPHRA_TX.toLowerCase(), "duplicate"),
```

Comment:
may be better to make the constant itself all lowercase so that people don't see something different in their annotation and what comes back from the rest api.

Reply:
Done.